execute/plugin_tokendata_test.go

```diff
@@ -87,31 +100,32 @@ func Test_USDC_Transfer(t *testing.T) {
 	require.Len(t, outcome.CommitReports, 1)
 
 	// Round 3 - Filter
-	// Messages 102-104 are executed, 105 doesn't have token data ready
+	// Messages 102-104,106,108 are executed, 105 and 107 don't have token data ready
 	outcome = runRoundAndGetOutcome(ctx, ocrTypeCodec, t, runner)
 	require.NoError(t, err)
-	require.Len(t, outcome.Report.ChainReports, 1)
+	assert.Len(t, outcome.Report.ChainReports, 1)
```

Comment:
Why these changes? `require` will return immeidately but `assert` will continue test execution. I think when number of reports and messages don't match there is no point in continuing